apps/ledger-live-desktop/src/renderer/components/RenderError.tsx

```diff
@@ -24,6 +25,7 @@ type Props = {
 export default function RenderError({ error, withoutAppData, children }: Props) {
   const { t } = useTranslation();
   const hardReset = useHardReset();
+  const troubleshootingCrashUrl = useLocalizedUrl(urls.troubleshootingCrash);
```

Comment:
❌ as discussed internally, we can't afford this. the RenderError must not access anything that needs the store, because if we have a crash in the store itself we would not even be able to render the error. so we must not localize this url in this precise case.
RenderError is a "critical error" case, that is basically a blue screen of death, and it remains in English.